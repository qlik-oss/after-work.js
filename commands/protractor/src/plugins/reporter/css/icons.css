@font-face {
	font-family: 'Sensei-Report';
	/*src: url('fonts/Sensei-Report.woff?qik77y') format('woff');*/
	src: url(data:application/font-woff;charset=utf-8;base64,d09GRgABAAAAAA+8AAwAAAAAD2wAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAABHU1VCAAABHAAAAUAAAAFA3ovhV09TLzIAAAJcAAAAYAAAAGAPEgehY21hcAAAArwAAADMAAAAzJMVql9nYXNwAAADiAAAAAgAAAAIAAAAEGdseWYAAAOQAAAI2AAACNh8eY+faGVhZAAADGgAAAA2AAAANgiqP+loaGVhAAAMoAAAACQAAAAkCKIE52htdHgAAAzEAAAAmAAAAJg3gAEYbG9jYQAADVwAAABOAAAAThx2GiBtYXhwAAANrAAAACAAAAAgAC0AcG5hbWUAAA3MAAABzgAAAc5CjNuFcG9zdAAAD5wAAAAgAAAAIAADAAAAAQAAAAoAHgAsAAFsYXRuAAgABAAAAAAAAAABAAAAAWxpZ2EACAAAAAEAAAABAAQABAAAAAEACgAAAAEAFAAHACYAYgB2AKQAxADWAPwAAQAHAAcACAAJAAwAEgAUACUAAwAIABQAKAAfAAUAEwARABQAFAAgAAkACwAJAAcADQAPAAUAEwANABkACQAFAA4ACQAQAAgABQATAAQAAQAEAB0ABwAMABQAEgAOAAUAGAADAAgAFgAiACQABgAPAAYACQAIAAQAIwAFAA8ABgAJAAgAHgAFAAUAEwAVAAsAAgAGABQAGwAGAA8ABQAKAAkAFAAaAAUADwAFAAoACQABAAQAIQAGAAUAFgAUAAkABAACAAYAGgAcAAkAFQARABIAFwAFABUABwALACIABQAMAAoADwAFAAEABAAkAAIAJQADA/YBkAAFAAACmQLMAAAAjwKZAswAAAHrADMBCQAAAAAAAAAAAAAAAAAAAAEQAAAAAAAAAAAAAAAAAAAAAEAAAOqBA8D/wABAA8AAQAAAAAEAAAAAAAAAAAAAACAAAAAAAAMAAAADAAAAHAABAAMAAAAcAAMAAQAAABwABACwAAAAKAAgAAQACAABACAAMgBlAGkAcAB1AHcAeekA6Q7pUulW6crqEOod6mfqgf/9//8AAAAAACAAMgBhAGcAawByAHcAeekA6Q3pUulW6crqD+od6mfqf//9//8AAf/j/9L/pP+j/6L/of+g/58XGRcNFsoWxxZUFhAWBBW7FaQAAwABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAH//wAPAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAAAAAAAAAIAADc5AQAAAAABAAAAAAAAAAAAAgAANzkBAAAAAAYAFv/EA+oDvAAOAB0ANABcAGYAbQAAARQGIyImPQE0NjMyFh0BJTQmIyIGHQEUFjMyNj0BBRUUBiMiJj0BIxUUBiMiJj0BIxEhESMBIiY3Mx4BMzI2NTQmIzUWNjU0JiMiBhcjPgEzMhYVFAYHHgEVFAYjJSM1IzUWNjczEQERIRUhESMCoxsSExoaExIb/m8bEhMaGhMSGwHHOikpOss6KSk6bAMsaf5zN0IBRQEXGhEdJiIYKhkOGBQBRgJALilFExEWGUcxAS9ITyYxBTsBGPzXA4BXAx4TGhoTcRIaGhJxcRIbGxJxExoaE3FCLik6OikuLSk6Oikt/RwC5P2uPj0bIxUXHgw2AgwYExIgGDs6MCsWIQgJJhkyOQbKOQEaJf6/Af39HVcDOgAAAAAEAAAAAAQAA4AAEAAiAC8ANgAAATgBMRE4ATEhOAExETgBMSE1ISIGFREUFjMhMjY1ETQmIzEHFAYjIiY1NDYzMhYVEyE1EwEzNwPA/IADgPyAGiYmGgOAGiYmGoA4KCg4OCgoOED9AOABAEDgA0D9AAMAQCYa/QAaJiYaAwAaJuAoODgoKDg4KP3ggAGA/sDAAAAABQAA/8AEgAPAABoALgA/AEwAUwAAASM1NCYjISIGFREUFjsBFRQWMyEyNjURNCYjBREjOAExETgBMSE4ATEVISIGFTEBOAExITgBMRE4ATEhOAExEQMUBiMiJjU0NjMyFhUTITUTATM3BEBAJhr8gBomJhpAJhoDgBomJhr8QEADgP0AGiYDwPyAA4CAOCgoODgoKDhA/QDgAQBA4ANAQBomJhr9ABomQBomJhoDABomQP2AAwBAJhr9AAMA/QACYCg4OCgoODgo/eCAAYD+wMAAAAACAED/wAOAA8AAIQBCAAABNTM1NCYrASIGHQEzFQ4DFRQeAjMyPgI1NC4CJxMOASMiJicuATU0Njc+ATcDBhYzMjYnAx4BFx4BFRQGBwIAgCUbwBslgFGMZzxBcZhWVphxQTxnjFHCLXVAQHUtLjAwLitwPRYBEg8PEgEWPXArLjAwLgL/QUAbJSUbQEEGRm+SUlaYcUFBcZhWUpJvRgb9fy4wMC4tdUBAdS0sMAL+xhQSEhQBOgIwLC11QEB1LQADAAAAAAQAA4AABAAJAA8AABMRIREhASERIREFIQ8BIScABAD8AAPA/IADgP7g/sAgQAIAQAOA/YACgP3AAgD+AICAQEAAAAACAAD/wAQAA8AAFABlAAABIg4CFRQeAjMyPgI1NC4CIxEiJicTPgE9ATQmIyIuAjUuASsBIgYdARQWHwEVLgM1NDY3MzI2PwE+AT0BPgEzMhYXDgEHDgEVFBYXHgEzOgEzHgIGBxQGFQ4DIwIAaruLUFCLu2pqu4tQUIu7ai9ZKekEBBMNKlRCKQUMBoANEwoIbixGMxsWFXUGDAWABAUeQSE1YywDBgMbHR0bHEYmAgUCBgsECxIBHklRWS8DwFCLu2pqu4tQUIu7amq7i1D8QBMRAQcECwZgDRMlLCUBBAUTDcAJEAQ3vB9PX2o5NGAsBQSABQwGTQkKFxYDBQMbRyYmRxsbHRNFX3dGAQMBIDEjEwABAAL/wgP+A74AVAAAJTgBMQkBOAExPgE3NiYvAS4BBw4BBzgBMQkBOAExLgEnJgYPAQ4BFx4BFzgBMQkBOAExDgEHBhYfAR4BNz4BNzgBMQkBOAExHgEXFjY/AT4BJy4BJwP3/skBNwIEAQMDB5MHEgkDBgL+yf7JAgYDCRIHkwcDAwEEAgE3/skCBAEDAweTBxIJAwYCATcBNwIGAwkSB5MHAwMBBAKJATcBNwIGAwkSB5MHAwMBBAL+yQE3AgQBAwMHkwcSCQMGAv7J/skCBgMJEgeTBwMDAQQCATf+yQIEAQMDB5MHEgkDBgIAAAABAAAAIAQAA0AABQAACQEnBwkBA2D+IOCgAYACgANA/iDgoP6AAoAAAgCAAEADgANAAAMABwAAEyERIQEhESGAAUD+wAHAAUD+wANA/QADAP0AAAABAAD/wAQAA8AAFgAAJTczAyE1CQE1IRMjJy4BIyEJASEyNjcDriwmQPxAAUz+tAPULCITGzY6/WoBYf7XAh5XQhXhX/6ASgGHAUzj/wAnNyL+n/6hOCkAAgAgAIAD4AMAAAUACwAAJRcJAQcXJScJATcnAkBgAUD+wGDg/qBg/sABQGDg4GABQAFAYODgYP7A/sBg4AAAAAMAIABXBOADKQAFAAsAEAAAJRcJAQcXJScJATcnARcDJxMDQGABQP7AYOD9oGD+wAFAYOAB3UbARsDgYAFAAUBg4OBg/sD+wGDgAWkS/UASAsAAAAEAAAAAAAAAAAACAAA3OQEAAAAAAQAAAAEAAKyIr/FfDzz1AAsEAAAAAADSVX22AAAAANJVfbYAAP/ABOADwAAAAAgAAgAAAAAAAAABAAADwP/AAAAFAAAAAAAE4AABAAAAAAAAAAAAAAAAAAAAJgQAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAWBAAAAASAAAAEAABABAAAAAQAAAAEAAACBAAAAAQAAIAEAAAABAAAIAUAACAAAAAAAAAAAAAKABQAHgAoADIAPABGAFAAWgBkAG4AeACCAIwAlgCgAKoAtAC+AMgA0gDcAOYA8AGIAdICPAKcAsADSgPCA9YD7AQYBDgEYgRsAAAAAQAAACYAbgAGAAAAAAACAAAAAAAAAAAAAAAAAAAAAAAAAA4ArgABAAAAAAABAA0AAAABAAAAAAACAAcAlgABAAAAAAADAA0ASAABAAAAAAAEAA0AqwABAAAAAAAFAAsAJwABAAAAAAAGAA0AbwABAAAAAAAKABoA0gADAAEECQABABoADQADAAEECQACAA4AnQADAAEECQADABoAVQADAAEECQAEABoAuAADAAEECQAFABYAMgADAAEECQAGABoAfAADAAEECQAKADQA7FNlbnNlaS1SZXBvcnQAUwBlAG4AcwBlAGkALQBSAGUAcABvAHIAdFZlcnNpb24gMS4wAFYAZQByAHMAaQBvAG4AIAAxAC4AMFNlbnNlaS1SZXBvcnQAUwBlAG4AcwBlAGkALQBSAGUAcABvAHIAdFNlbnNlaS1SZXBvcnQAUwBlAG4AcwBlAGkALQBSAGUAcABvAHIAdFJlZ3VsYXIAUgBlAGcAdQBsAGEAclNlbnNlaS1SZXBvcnQAUwBlAG4AcwBlAGkALQBSAGUAcABvAHIAdEZvbnQgZ2VuZXJhdGVkIGJ5IEljb01vb24uAEYAbwBuAHQAIABnAGUAbgBlAHIAYQB0AGUAZAAgAGIAeQAgAEkAYwBvAE0AbwBvAG4ALgAAAAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=) format('woff');
	font-weight: normal;
	font-style: normal;
}

i, .icomoon-liga {
	font-family: 'Sensei-Report';
	speak: none;
	font-style: normal;
	font-weight: normal;
	font-variant: normal;
	text-transform: none;
	line-height: 1;

	/* Enable Ligatures ================ */
	letter-spacing: 0;
	-webkit-font-feature-settings: "liga";
	-moz-font-feature-settings: "liga=1";
	-moz-font-feature-settings: "liga";
	-ms-font-feature-settings: "liga" 1;
	-o-font-feature-settings: "liga";
	font-feature-settings: "liga";
}

.sensei-calendar:before {
	content: "\e900";
}
.sensei-image:before {
	content: "\e90d";
}
.sensei-images:before {
	content: "\e90e";
}
.sensei-stopwatch:before {
	content: "\e952";
}
.sensei-display:before {
	content: "\e956";
}
.sensei-earth:before {
	content: "\e9ca";
}
.sensei-cross:before {
	content: "\ea0f";
}
.sensei-checkmark:before {
	content: "\ea10";
}
.sensei-pause2:before {
	content: "\ea1d";
}
.sensei-sigma:before {
	content: "\ea67";
}
.sensei-embed:before {
	content: "\ea7f";
}
.sensei-embed2:before {
	content: "\ea80";
}
