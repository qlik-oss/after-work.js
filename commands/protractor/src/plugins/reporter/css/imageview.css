.image-viewer-container {
    color: #666;
    position: fixed;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
    background: rgba(51, 51, 51, 0.8);
    z-index: 100;
    display: none;
    justify-content: center;
    align-items: center;
    opacity: 0;
    transition: opacity 0.2s ease;
}
.image-viewer {
    background: white;
    box-shadow: 2px 2px 16px rgba(0, 0, 0, 0.1);
    border-radius: 4px;
    position: relative;
    transform: scale(0.8);
    transition: transform 0.4s ease;
}
.image-viewer-container.visible {
    opacity: 1;
}
.image-viewer-container.visible .image-viewer {
    transform: scale(1);
}
.image-viewer img,
.image-viewer h4 {
    padding: 1em;
    margin: 0;
}
.image-viewer h4 {
    background: #fafafa;
    border-radius: 4px 4px 0 0;
}
.image-viewer .baseline,
.image-viewer .diff {
    display: none;
}
.image-viewer .content:hover .baseline {
    display: block;
}
.image-viewer .content:hover .regression {
    display: none;
}
.image-viewer .content:active .diff {
    display: block;
}
.image-viewer .content:active .regression,
.image-viewer .content:active .baseline {
    display: none;
}
.close {
    position: absolute;
    top: -1.5em;
    right: -1.5em;
    cursor: pointer;
    border-radius: 50%;
    color: white;
    background: #666;
    border: 2px solid white;
    padding: 0.3em 0.5em;
    line-height: 1em;
}
