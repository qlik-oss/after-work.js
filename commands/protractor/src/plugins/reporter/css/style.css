body{
    font-family: "Roboto", sans-serif;
    font-size: 14px;
    text-rendering: optimizeLegibility !important;
    -webkit-font-smoothing: antialiased !important;
    -moz-osx-font-smoothing: grayscale !important;
    line-height: 1.42857143;
    color: #333;
    background-color: #fff;
    margin: 0px;
}
a{
    text-decoration: none;
    color: inherit;
}
pre{
    margin-top: 0px;
}
h1, h1, h3, h4 {
    text-shadow:1px 1px 1px 1px rgba(0,0,0,0.005);
    margin: 0px;
}
li{
    list-style-type:none;
}
.top_header{
    height: 40px;
    padding-left: 50px;
    padding-right: 50px;
    background: #333;
    color: #eee;
    -webkit-box-shadow: rgba(0,0,0,0.8) 0px 0px 32px;
    -moz-box-shadow: rgba(0,0,0,0.8) 0px 0px 32px;
    box-shadow: rgba(0,0,0,0.8) 0px 0px 32px;
}
.name{
    height: 22px;
}
.version{
    font-size: 8px;
}
.description{
    color: #bbb;
    font-size: 10px;
    text-transform: uppercase;
}
.info_header{
    display: flex;
    flex-flow: row;
    justify-content: space-between;
    align-content: center;
    align-items: stretch;
    height: 30px;
    line-height: 30px;
    padding-left: 50px;
    padding-right: 50px;
    font-size: 10px;
    text-transform: uppercase;
    border-bottom: 1px solid rgba(0,0,0,0.25);
    -webkit-box-shadow: rgba(255,255,255,0.25) 0px 1px 0px;
    -moz-box-shadow: rgba(255,255,255,0.25) 0px 1px 0px;
    box-shadow: rgba(255,255,255,0.25) 0px 1px 0px;
}
.info_header ul
{
    margin: 0px;
    padding: 0px;
    flex: 1 1 auto;
}
.info_header li
{
    display: inline;
    list-style-type: none;
    padding-left: 10px;
    padding-right: 10px;
    border-right: 1px solid rgba(0,0,0,0.25);
    -webkit-box-shadow: rgba(255,255,255,0.25) 1px 0px 0px;
    -moz-box-shadow: rgba(255,255,255,0.25) 1px 0px 0px;
    box-shadow: rgba(255,255,255,0.25) 1px 0px 0px;
}
.info li:first-child,
.stats li:last-child
{
    border-left: 1px solid rgba(0,0,0,0.25);
    -webkit-box-shadow: inset rgba(255,255,255,0.25) 1px 0px 0px;
    -moz-box-shadow: inset rgba(255,255,255,0.25) 1px 0px 0px;
    box-shadow: inset rgba(255,255,255,0.25) 1px 0px 0px;
}
.info li{
    float: left;
}
.stats li{
    float: right;
}
.status i{
    padding-right: 4px;
}
/*.stats li:before {
    content: " ";
    display: block;
    background: #c00;
    height: 2px;
    width: 100%;
    right:0px;
    top: 0px;
}*/
.info_header i{
    padding-right: 4px;
}
.passed{
    color: #5cb85c;
}
.failed{
    color: #d9534f;
}
.pending{
    color: #5bc0de;
}
.total{
    color: magenta;
}
.tests{
    padding: 50px;
}
.collapsed{
    display: none;
}
.message{
    padding-left: 25px;
    padding-right: 5px;
}
.tests h4{
    font-weight: 400;
    display: inline-block;
}
.testcase{
    display: flex;
    flex-flow: row;
    justify-content: space-between;
    align-content: center;
    align-items: stretch;
    padding-top: 3px;
}
.testName{
    flex: 1 1 auto;
}
.testInfo{
    flex: 0 1 auto;
}
.hljs{
    font-size: 85%;
    border-radius: 4px;
}
article:after{
    content: " ";
    display: block;
    background: #ccc;
    height: 1px;
    width: 100%;
    right:0px;
    top: 0px;
}
