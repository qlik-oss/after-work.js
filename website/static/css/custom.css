/* your custom css */
.homeContainer, .fixedHeaderContainer {
  min-height: 50px;
  background: linear-gradient(90deg, #FFD194 0%, #D1913C 100%);
}

.navigationSlider .slidingNav ul li a[href^="http"] {
  font-family: FontAwesome;
  font-size: 1.3em;
  padding: 0 5px;
  margin-left: 5px;
}

.projectLogo {
  display: block;
}

footer.nav-footer {
  background: #5d5d5d;
}

footer .sitemap {
  max-width: 680px;
}

.container .wrapper h2 {
  font-variant: small-caps;
  font-weight: 200;
}

.container .gridBlock .blockContent p {
  text-align: left;
  color: #666;
}

@media only screen and (min-width: 1200px) {
  .homeContainer .homeWrapper .projectLogo {
    padding: 2em 0px 4em;
  }
}

.homeContainer .button {
  color: #e8ffe9;
  border: 0;
  background: #d1913c;
  border-radius: 4px;
  padding: 1em 2em;
  box-shadow: 0 0px 8px rgba(0, 50, 0, 0.1);
  transition: all 0.1s ease-out;
  position: relative;
  top: 0;
}

.homeContainer .button.scream {
  background: linear-gradient(90deg, rgba(255, 107, 63, 0.85) 0%, rgba(255, 59, 114, 0.85) 100%)
}

.homeContainer .button:hover {
  box-shadow: 0 6px 18px rgba(0, 50, 0, 0.1);
  top: -1px;
}

.homeContainer::before {
  display: block;
  content: '';
  height: 402px;
  background: linear-gradient(90deg, #FFD194 0%, #D1913C 100%);
  z-index: -1;
  position: absolute;
  width: 100%;
}

.homeContainer .homeWrapper {
  padding: 3em 10px 0 10px
}

.homeContainer .homeWrapper .logo {
  opacity: 0.8;
}

.projectTitle {
  color: #ffffff;
  opacity: 0.8;
}

.projectTitle > small {
  margin-bottom: 0;
}

.promoSection {
  top: 1em;
}

