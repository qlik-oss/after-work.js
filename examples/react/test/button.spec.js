import React from 'react';
import renderer from 'react-test-renderer';
import 'foo.scss'; // eslint-disable-line import/no-unresolved
import 'bar.less'; // eslint-disable-line import/no-unresolved
import 'baz.css'; // eslint-disable-line import/no-unresolved
import Button from '../src/button';

describe('button', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Button>Text</Button>).toJSON();
    expect(tree).toMatchSnapshot();
    const tree1 = renderer.create(<Button>Text1</Button>).toJSON();
    expect(tree1).toMatchSnapshot();
  });
  it('renders fancy', () => {
    const tree1 = renderer.create(<Button>fancy1</Button>).toJSON();
    expect(tree1).toMatchSnapshot();
  });
});
