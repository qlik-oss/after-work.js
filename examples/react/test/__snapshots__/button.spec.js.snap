// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'renders correctly' 1`] = `
<button>
  Text
</button>
`;

exports[`'renders correctly' 2`] = `
<button>
  Text1
</button>
`;

exports[`'renders correctly' 3`] = `
<button>
  Text
</button>
`;

exports[`'renders correctly' 4`] = `
<button>
  Text1
</button>
`;

exports[`'renders correctly' 5`] = `
<button>
  Text
</button>
`;

exports[`'renders correctly' 6`] = `
<button>
  Text1
</button>
`;

exports[`'renders fancy' 1`] = `
<button>
  fancy1
</button>
`;

exports[`'renders fancy' 2`] = `
<button>
  fancy1
</button>
`;

exports[`'renders fancy' 3`] = `
<button>
  fancy1
</button>
`;
